index.js

```diff
@@ -88,7 +92,18 @@ module.exports = function(babel) {
   function compileTemplate(precompile, template, _options) {
     let options = Object.assign({ contents: template }, _options);
 
-    let precompileResult = precompile(template, options);
+    let precompileResult;
+
+    if (options.insertRuntimeErrors) {
+      try {
+        precompileResult = precompile(template, options);
+      } catch (error) {
+        return runtimeErrorIIFE({ ERROR_MESSAGE: error.message });
```

Comment:
I assume babel is taking care of escaping for us, is that right?

Reply:
not sure, will test

Reply:
Ya, the `error.message` itself is already a string, and any valid string will work (including any escaping needed internally).